packages/victory-core/src/victory-util/selection.ts

```diff
@@ -1,6 +1,7 @@
 import React from "react";
 import { Collection, VictoryCommonProps } from "./index";
 import type { DomainPropType, ScaleXYPropType } from "../types/prop-types";
+import { Tuple } from "../../lib";
```

Comment:
We probably shouldn't import from `lib` like this, probably a VS Code auto-mistake

Reply:
Oops, dang you auto-import 😅